components/core/Profile.js

```diff
@@ -508,7 +523,10 @@ export default class Profile extends React.Component {
           <div css={STYLES_PROFILE_INFO}>
             <div
               css={STYLES_PROFILE_IMAGE}
-              style={{ backgroundImage: `url('${creator.data.photo}')` }}
+              style={{
+                backgroundImage: `url('${creator.data.photo}')`,
+                borderColor: this.state.isOnline ? `green` : `white`,
```

Comment:
Lets make sure we use constants values for this, I know its tedious but keeps it sane.